### PR TITLE
docs: answer question about recommendation system

### PR DESCRIPTION
**Pull request by MiaaaChen**
Answer question about recommendation system

### CODE REVIEW

**Comment by Tonyrj3268** [comment]
妳的part感覺比較重要誒，要不要把妳的順序放到前面去

**Comment by MiaaaChen** [comment]
好！我改一下

File: planning-01/readme.md [hunk 1]
```diff
@@ -2,33 +2,79 @@
...
+
+### 3. 混合模型
+
+-   推薦邏輯：由於不同用戶的資料量和行為習慣不同，我們可以將基於內容和協同過濾的推薦系統進行混合，這樣能夠根據具體情況靈活選擇最佳的推薦方式。
+
+-   推薦策略：
+
+    -   當用戶的行為資料不足時，使用基於內容的推薦。
+    -   當用戶資料充足時，使用協同過濾來提高推薦準確性。
+    -   引入隨機推薦或多樣性推薦來打破回聲室效應（Echo Chamber Effect）。
```
[the remark this answers is not in the document]
Comment (Tonyrj3268) [reply planning-01/readme.md:37]: 看起來好像兩個回都可以，應該可以不改

**Review by Tonyrj3268 — APPROVED** [review]
lgtm

**Pull request merged**